app/views/proprietor/accounts/edit.html.erb

```diff
@@ -34,6 +34,32 @@
             <%= s.input :base_path %>
           <% end %>
 
+          <p>
+						<%= f.check_box :shared_search, {checked: @account.is_shared_search_enabled?, name: "account[settings][shared_search]", id: "account_settings_shared_search"}, 'true' , 'false' %>
+						<%= f.label "Search Only(select tenant on next page)" %>
```

Comment:
please use i18n tags

Reply:
Thanks I will use i18n tags in the update